app/src/main/java/com/helio/app/MainActivity.java

```diff
@@ -28,5 +38,31 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // This makes the fragment change when you press the navigation buttons
         NavigationUI.setupWithNavController(navView, navController);
+
+        fetchState();
+    }
+
+    private void fetchState() {
```

Comment:
Can you add a comment/javadoc to explain what this method does?
(or will do, it is basically debug at the moment it looks like)

Reply:
It is indeed just some controller code to make some requests for the time being. It will be removed when we connect the client code to UI actions.